deploy/codebase/updateinit.js

```diff
@@ -72,7 +94,7 @@ async.series([
   async.apply(command, `cp -PR /tmp/extract/package/subpackage/lamassu-machine ${applicationParentFolder}`),
   async.apply(command, `cp -PR /tmp/extract/package/subpackage/hardware/${hardwareCode}/node_modules ${applicationParentFolder}/lamassu-machine/`),
   async.apply(installDeviceConfig),
-  async.apply(command, 'systemctl enable supervisor'),
```

Comment:
Why is this line deleted?

Reply:
Its something from another PR, fixed on rebase.